contracts/Whitelist.sol

```diff
@@ -59,12 +96,33 @@ contract Whitelist is Ownable {
     }
 
     /**
-     * @notice set a product hash as supported
+     * @notice allow Otoken Factory to whitelist option
+     * @dev can only be called from owner address
+     * @param _otokenAddress otoken
+     */
+    function whitelistOtoken(address _otokenAddress) external onlyFactory {
```

Comment:
Didn't we change the name to `registerOtoken` already?

Reply:
I saw your comments on the interface thread, but I think we better discuss this more.